doc/source/drivers/vector/mssqlspatial.rst

```diff
@@ -57,7 +57,13 @@ to select the proper database.
 
 The connection may contain the optional **Driver** parameter if a custom
 SQL server driver should be loaded (like FreeTDS). The default is **{SQL
-Server}**
+Server}**.
+
+Authentication is supported either through a **TrustedConnection** or
+through username (**UID**) and password (**PWD**). As providing username
+and password on the commandline can be a security issue, login credentials
+can also be more securely stored in user defined environment variables
+*MSSQLSPATIAL_UID* and *MSSQLSPATIAL_PWD*.
```

Comment:
I'd suggest to use GDAL-specific names `GDAL_MSSQLSPATIAL_UID` and `GDAL_MSSQLSPATIAL_PWD`

Reply:
Well, existing config options https://gdal.org/drivers/vector/mssqlspatial.html#configuration-options all starts with ``MSSQLSPATIAL_``, so better keep with that schem

Reply:
The docs call these two environment  variables, but I guess they are not distinguished from config options

Reply:
they are queried with CPLGetConfigOption(), so can be set through env. variable or config options

Reply:
I` ll go with whatever name for the variables you prefer...